docs/docs/configuration/materializations/iceberg.md

```diff
@@ -46,36 +46,34 @@ Iceberg supports several **table formats** for data : `PARQUET`, `AVRO` and `ORC
 
 It is possible to use iceberg in an incremental materialization. Two strategies are supported:
 
-- `append`: new records are appended to the table, this can lead to duplicates
-- `merge`: must be used in combination with `unique_key`.
-  It performs an upsert: new records are added, and records that already existed, are updated. If
-  `delete_condition` is provided in the model config, it can also delete records based on the
-  provided condition (SQL condition). You can use any column of the incremental table (`src`) or
-  the final table (`target`). You must prefix the column by the name of the table to prevent
-  `Column is ambiguous` error.
+- `append`: New records are appended to the table, this can lead to duplicates.
+- `merge`: Performs an upsert (and optional delete), where new records are added and existing records are updated. Only available with Athena engine version 3.
+    - `unique_key` **(required)**: columns that define a unique record in the source and target tables.
+    - `incremental_predicates` (optional): SQL conditions that enable custom join clauses in the merge statement. This can be useful for improving performance via predicate pushdown on the target table.
```

Comment:
It would be nice to put a block to say that this is available since 1.5.1. You can check examples elsewhere in the documentation

Reply:
I added the blocks but not sure I did it correctly. How do I do a full preview of the docs with the version drop-down?

Reply:
@chrischin478 sorry for late reply from us. Seems `npm run start` allows you to run docusaurus locally.

Reply:
@chrischin478 I've created PR which fixes your changes and adds extra infor about merge update condition. @Jrmyy @nicor88 pls review: https://github.com/dbt-athena/dbt-athena.github.io/pull/13. So we can merge only it or these 2 PRs consecutively.